### PR TITLE
qbittorrent-portable: add

### PR DESCRIPTION
**Pull request by Skrity**

### CODE REVIEW

**Comment by Ash258** [comment]
I would wait for one of these is merged: https://github.com/qbittorrent/qBittorrent/pull/9584 https://github.com/qbittorrent/qBittorrent/pull/9349 and then add qbittorrent-portable to extras, keeping nonportable available

File: bucket/qbittorrent.json [hunk 1]
```diff
@@ -16,14 +16,22 @@
             "hash": "72ecfe934a8dc819f47bb100011b9bb3569b4e9ee2d1fbe7cd1d048662b2198b"
         }
     },
-    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
```
Comment (Ash258) [inline bucket/qbittorrent.json]: Remove what you added and keep only persist and this:
```suggestion
    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Force -Recurse",
```
Check for persited folder is redundant

**Comment by Skrity** [comment]
I see merit in keeping qbittorrent intact if only for compatibility with previous installations.
As for portability: I imagine that env_set will break installation of qbittorrent alongside qbittorrent-portable, maybe something like this can work meanwhile we wait for a proper portable edition? 
```
    "bin": [ 
        [
            "qbittorrent.exe",
            "qbittorrent",
            "--portable" 
        ]
    ],
    "shortcuts": [
        [
            "qbittorrent.exe",
            "qBittorrent",
            "--portable" 
        ]
    ],
```

**Comment by Ash258** [comment]
I use this method inside my bucket:
https://github.com/Ash258/scoop-Ash258/blob/master/bucket/qBittorrent.json

File: bucket/qbittorrent-portable.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+    "homepage": "https://www.qbittorrent.org/",
+    "license": {
```
Comment (Ash258) [inline bucket/qbittorrent-portable.json]:
```suggestion
    "license": "GPL-2.0-only",
```

File: bucket/qbittorrent-portable.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        "url": "https://www.qbittorrent.org/download.php",
+        "re": "Latest:\\s+v([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://downloads.sourceforge.net/project/qbittorrent/qbittorrent-win32/qbittorrent-$version/qbittorrent_$version_x64_setup.exe#/dl.7z",
+                "hash": {
+                    "url": "https://www.qbittorrent.org/download.php",
+                    "find": "64-bit.*\\s+.*<code>([a-fA-F0-9]{40,128})</code>"
```
Comment (Ash258) [inline bucket/qbittorrent-portable.json]:
```suggestion
                    "regex": "64-bit.*\\s+.*<code>$sha256</code>"
```

File: bucket/qbittorrent-portable.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                }
+            },
+            "32bit": {
+                "url": "https://downloads.sourceforge.net/project/qbittorrent/qbittorrent-win32/qbittorrent-$version/qbittorrent_$version_setup.exe#/dl.7z",
+                "hash": {
+                    "url": "https://www.qbittorrent.org/download.php",
+                    "find": "32-bit.*\\s+.*<code>([a-fA-F0-9]{40,128})</code>"
```
Comment (Ash258) [inline bucket/qbittorrent-portable.json]:
```suggestion
                    "regex": "32-bit.*\\s+.*<code>$sha256</code>"
```

File: bucket/qbittorrent.json [hunk 5]
```diff
@@ -16,22 +16,14 @@
             "hash": "72ecfe934a8dc819f47bb100011b9bb3569b4e9ee2d1fbe7cd1d048662b2198b"
         }
     },
-    "pre_install": "if(!(Test-Path \"$persist_dir\\profile\")) { New-Item \"$dir\\profile\" -Type Directory | Out-Null }",
-    "persist": "profile",
-    "post_install": [
-        "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
-        "Remove-Item \"$dir\\uninst.exe\" -Force"
-    ],
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
```
Comment (Ash258) [inline bucket/qbittorrent.json]:
```suggestion
    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\uninst.exe\" -Force -Recurse",
```

**Pull request merged**